tests/test_scripts.py

```diff
@@ -2693,6 +2693,17 @@ def test_trim_low_abund_trimtest_savetable():
                 'GGTTGACGGGGCTCAGGGGGCGGCTGACTCCGAGAGACAGCA'
 
 
+def test_counting():
```

Comment:
On Wed, Feb 25, 2015 at 08:38:25AM -0800, Michael R. Crusoe wrote:
> > @@ -2693,6 +2693,17 @@ def test_trim_low_abund_trimtest_savetable():
> >                  'GGTTGACGGGGCTCAGGGGGCGGCTGACTCCGAGAGACAGCA'
> >  
> >  
> > +def test_counting():
> 
> @ctb If the counting file format changes then shouldn't this test fail?

that's a test with a different purpose, I think.

--titus